helm/pms-partner/values.yaml

```diff
@@ -47,7 +47,7 @@ service:
 image:
   registry: docker.io
   repository: mosipid/partner-management-service
```

Comment:
repository: mosipid/partner-management-service to repository: mosipqa/partner-management-service